python/hsfs/core/transformation_function_engine.py

```diff
@@ -30,6 +31,13 @@ class TransformationFunctionEngine:
         "robust_scaler",
         "label_encoder",
     ]
+    AMBIGUOUS_FEATURE_ERROR = (
+        "Provided feature '{}' is ambiguous and exists in more than one feature groups. "
+        "You can provide the feature with the prefix you specify in the join."
+    )
+    FEATURE_NOT_EXIST_ERROR = (
+        "Provided feature '{}' do not exist in any of the feature groups."
```

Comment:
We should make the error message more specific. And need to change the one [here](https://github.com/logicalclocks/feature-store-api/pull/836/files#diff-fefd147df0a678283fbf909e864223aba195e25f21cd61a44258cc8ed1c83a0eR46) as well.